models/helper.go

```diff
@@ -16,10 +16,10 @@ func toRadians(angleInDegrees float64) (angleInRadians float64) {
   return
 }
 
-// TODO: this function needs to be destroyed, along with refactoring to not have Location and UserLocation as separate things
+// TODO: this function needs to be destroyed, along with refactoring to not have Location and Location as separate things
 func CalculateLocationDistance(location1 Location, location2 Location) (distance float64){
-  userLocation1 := UserLocation{Latitude : location1.Latitude, Longitude : location1.Longitude}
-  userLocation2 := UserLocation{Latitude : location2.Latitude, Longitude : location2.Longitude}
-  distance = CalculateDistance(userLocation1, userLocation2)
+  Location1 := Location{Latitude : location1.Latitude, Longitude : location1.Longitude}
```

Comment:
These are already Locations now, so you can just `CalculateDistance` instead of using this func

Reply:
yup realized that too, pushed a fresh diff to take care of it